addons/main.tf

```diff
@@ -1,3 +1,17 @@
+terraform {
```

Comment:
Remove `terraform` and `provider` blocks. Both are taken care of elsewhere (with the `workspace` and in the `provider.tf` file (added in PR #25 ) respectively).